plugins/filter_kubernetes/kubernetes.c

```diff
@@ -225,22 +225,24 @@ static int pack_map_content(msgpack_packer *pck, msgpack_sbuffer *sbuf,
     void *log_buf = NULL;
     size_t log_size = 0;
     msgpack_unpacked result;
-    msgpack_object k;
-    msgpack_object v;
+    msgpack_object *k;
+    msgpack_object *v;
```

Comment:
Why do you switch to pointers?

Reply:
I re-used code from the `out_es` plugin, which is using pointers. I tried to make something consistent but I guess we can implement this feature without using pointers. I can switch back.

Reply:
I don't see the benefit of switching to pointers: we should stay without pointers IMO.